cfg/config.go

```diff
@@ -3,11 +3,84 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+	f, err := os.Open(".env")
+
+	if err != nil {
+		log.Printf("%s", err)
```

Comment:
You don't need to do any prints here. If you want to log something important - do it in Run() function.

Reply:
Fixed